server/src/routes/router.rs

```diff
@@ -0,0 +1,37 @@
+// skeleton for GQL router
```

Comment:
This isn't necessary. I would remove this

Reply:
this file is not included in new draft PR: https://github.com/Granola-Team/mina-on-chain-voting/pull/237